iOS/Flows/Message Detail/MessageDetailViewController.swift

```diff
@@ -96,20 +96,26 @@ class MessageDetailViewController: DiffableCollectionViewController<MessageDetai
     override func retrieveDataForSnapshot() async -> [MessageDetailDataSource.SectionType : [MessageDetailDataSource.ItemType]] {
         var data: [MessageDetailDataSource.SectionType : [MessageDetailDataSource.ItemType]] = [:]
         
-        data[.options] = [.option(.viewReplies), .option(.pin), .option(.edit), .option(.more)].reversed()
         
         guard let msg = ChatClient.shared.messageController(for: self.message)?.message else { return data }
         
-        data[.reads] = msg.readReactions.filter({ reaction in
+        data[.options] = [.option(.viewReplies), .option(.pin), .option(.edit), .more(MoreOptionModel(message: msg, option: .more))].reversed()
+
+        
+        let reads:[MessageDetailDataSource.ItemType] = msg.readReactions.filter({ reaction in
             return !reaction.author.isCurrentUser
         }).compactMap({ read in
-            return .read(read)
+            return .read(ReadViewModel(readReaction: read))
         })
         
-        if msg.replyCount > 0 {
-            data[.recentReply] = [.reply(msg)]
+        if reads.isEmpty {
+            data[.reads] = [.read(ReadViewModel(readReaction: nil))]
```

Comment:
I have to look into why the section background is being displayed, even though not items are in it, so for now I add an empty cell.